Makefile

```diff
@@ -84,8 +85,14 @@ else
 TESTCMD = -run $(RUNTEST)
 endif
 
+test_skip_kube: SKIP_KUBE=1
```

Comment:
One of the primary design aspects of Make is to augment its behavior by overriding variables. If we were to follow this pattern you've started here, things would get unwieldy rather quickly. This new target is unnecessary as you could achieve the same thing by running:

```
SKIP_KUBE=1 make test
```